programmers/난이도별/level01.핸드폰_번호_가리기/HyeonJeong.py

```diff
@@ -0,0 +1,5 @@
+def solution(phone_number):
+    answer = ""
+    for i, c in enumerate(phone_number):
+        answer += "*" if i < len(phone_number) - 4 else c
```

Comment:
```suggestion
    answer = phone_number.replace(phone_number[0:-4],'*'*len(phone_number[0:-4]))
```

```replace(a, b)```는 문자열의 a를 b로 바꾸어주는 함수입니다. ```replace()```함수와 문자열 슬라이싱을 사용하면 반복문을 사용하지 않으셔도 됩니다!

Reply:
replace()를 이용해 코드를 고쳐 보겠습니다. 감사합니다!!